demo/src/test/java/com/example/test/core/helper/ReadConfigPropertyFile.java

```diff
@@ -5,7 +5,7 @@
 
 public class ReadConfigPropertyFile {
 
-    public static String propertiesFilePath ="src\\test\\java\\com\\example\\test\\core\\configs.properties";
+    public static String propertiesFilePath ="src//test//java//com//example//test//core//configs.properties";

```

Comment:
pls change

Reply:
Have changed to be suitable for window user